src/Lkl.cc

```diff
@@ -840,7 +856,12 @@ void Lkl::SetParameters(const Char_t** parname, Double_t* parstart, Double_t* pa
       fParErr[ipar]     = 0;
       fIsParFixed[ipar] = kFALSE;
       if(fMinuit)
-	fMinuit->DefineParameter(ipar, fParName[ipar], fParStart[ipar], fParDelta[ipar], 0, 0);
```

Comment:
Same comment as above, related to the special treatment for all parameters called "tau"

Reply:
Agreed!